pkgs/by-name/ib/ibm-plex/package.nix

```
@@ -1,28 +1,59 @@
-{ lib, stdenvNoCC, fetchzip }:
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  families ? [ ],
+}:
+let
+  version = import ./version.nix;
+  availableFamilies = import ./hashes.nix;
 
-stdenvNoCC.mkDerivation rec {
+  selectedFamilies =
+    if (families == [ ]) then
+      (builtins.attrNames availableFamilies)
+    else
+      let
+        unknown = lib.subtractLists (builtins.attrNames availableFamilies) families;
+      in
+      if (unknown != [ ]) then throw "Unknown font(s): ${lib.concatStringsSep " " unknown}" else families;
+in
+stdenvNoCC.mkDerivation {
```

it would be better to `assert` this
```suggestion
  availableFamilyNames = builtins.attrNames availableFamilies;

  selectedFamilies =
    if (families == [ ]) then
      availableFamilyNames
    else
      families;
      
  unknownFamilies = lib.substractLists availableFamilynames families;
in
assert lib.assertMsg (unknownFamilies == [ ]) "Unknown font(s): ${toString unknownFamilies";
stdenvNoCC.mkDerivation {
```